examples/camera-app/linux/src/clusters/webrtc-provider/webrtc-provider-manager.cpp

```diff
@@ -352,6 +349,8 @@ CHIP_ERROR WebRTCProviderManager::HandleProvideICECandidates(uint16_t sessionId,
         }
     }
 
+    ScheduleICECandidatesSend();
```

Comment:
It seems the camera device only sends its ICE candidates after it receives ICE candidates from the controller. Therefore, the controller must always send its ICE candidates promptly after the SDP exchange.

Reply:
Thank you for your review, Yes, in order to simplify the exchange of ice-candidates outside of an automated flow, I have kept it this way so that, Camera can send ice-candidates as soon as it receives the same from the other peer.